src/app/components/annotations/pages/annotationSearchParameters.ts

```diff
@@ -45,13 +44,12 @@ export interface IAnnotationSearchParameters {
   time: MonoTuple<Duration, 2>;
 }
 
+// we exclude project, region, and site from the serialization table because
+// we do not want them emitted in the query string
```

Comment:
we don't?

seems like a bug waiting to happen

I'm inferring that we take these values from route context?

Reply:
Will be fixed when I resolve

https://github.com/QutEcoacoustics/workbench-client/pull/2145#discussion_r1802117157